### PR TITLE
Replace 'existsRole' query

### PR DESCRIPTION
**Pull request by webrouse**
Jira: https://keboola.atlassian.net/browse/COM-82

### CODE REVIEW

File: src/Connection.php [hunk 1]
```diff
@@ -129,19 +129,16 @@ public function fetchAll(string $sql, array $bind = []): array
...
-    public function fetchRoles(?string $roleLike = null): array
+    public function existsRole(string $role): bool
     {
-        $sql = 'SELECT * FROM INFORMATION_SCHEMA.APPLICABLE_ROLES';
-        $args = [];
-        if ($roleLike !== null) {
-            $sql .= ' WHERE ROLE_NAME = %s';
-            $args[] = $this->quote($roleLike);
-        }
+        $sql = 'SELECT COUNT(*) AS CNT FROM INFORMATION_SCHEMA.ENABLED_ROLES  WHERE ROLE_NAME = %s';
```
Comment (webrouse) [inline src/Connection.php:134]: Query `INFORMATION_SCHEMA.APPLICABLE_ROLES` stale pada na SNFK incidente, ... support to riesi, ale nejak sme sa nepohli zatial, .... preto som hladal ine riesenia, a zistil som, ze v tomto pripade mozeme pouzit aj `INFORMATION_SCHEMA.ENABLED_ROLES`, ktora ide dobre. Klient na to caka, tak to chcem spravit takto.

File: src/Connection.php [hunk 2]
```diff
@@ -129,19 +129,16 @@ public function fetchAll(string $sql, array $bind = []): array
         }
     }
 
-    public function fetchRoles(?string $roleLike = null): array
+    public function existsRole(string $role): bool
```
Comment (webrouse) [inline src/Connection.php:132]: Nepotrebujeme `fetchRoles` staci nam kontrolovat ci rola existuje.

**Comment by webrouse** [comment]
Otestoval som to aj na produkcii (tag `test3.7`):
1. job: https://connection.eu-central-1.keboola.com/admin/projects/82/jobs/229370325
image


2. job: https://connection.eu-central-1.keboola.com/admin/projects/82/jobs/229371606
image

A spravne je nastaveny aj timeout, co je ciel:
image

**Review by ondrajodas — APPROVED** [review]
ok

**Pull request merged**